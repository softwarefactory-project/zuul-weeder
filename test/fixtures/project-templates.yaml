- project-template:
    name: sf-ci-jobs
    check:
      jobs:
        - sf-rpm-build
        - sf-ci-functional-minimal:
            dependencies:
              - sf-rpm-build
        - sf-ci-functional-allinone:
            dependencies:
              - sf-rpm-build
        - sf-tenants:
            dependencies:
              - sf-rpm-build
    gate:
      queue: sf-master
      jobs:
        - sf-rpm-build
        - sf-ci-functional-minimal:
            dependencies:
              - sf-rpm-build
        - sf-ci-functional-allinone:
            dependencies:
              - sf-rpm-build
        - wait-for-changes-ahead:
            dependencies:
              - sf-ci-functional-minimal
              - sf-ci-functional-allinone
