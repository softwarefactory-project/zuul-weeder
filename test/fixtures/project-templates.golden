[ ZProjectTemplate
    ( ProjectTemplate
        { name = ProjectTemplateName "sf-ci-jobs"
        , pipelines = fromList
            [ ProjectPipeline
                { name = PipelineName "check"
                , jobs =
                    [ PJName
                        ( JobName "sf-rpm-build" )
                    , PJJob
                        ( Job
                            { name = JobName "sf-ci-functional-minimal"
                            , parent = Nothing
                            , nodeset = Nothing
                            , branches = []
                            , dependencies =
                                [ JobName "sf-rpm-build" ]
                            }
                        )
                    , PJJob
                        ( Job
                            { name = JobName "sf-ci-functional-allinone"
                            , parent = Nothing
                            , nodeset = Nothing
                            , branches = []
                            , dependencies =
                                [ JobName "sf-rpm-build" ]
                            }
                        )
                    , PJJob
                        ( Job
                            { name = JobName "sf-tenants"
                            , parent = Nothing
                            , nodeset = Nothing
                            , branches = []
                            , dependencies =
                                [ JobName "sf-rpm-build" ]
                            }
                        )
                    ]
                }
            , ProjectPipeline
                { name = PipelineName "gate"
                , jobs =
                    [ PJName
                        ( JobName "sf-rpm-build" )
                    , PJJob
                        ( Job
                            { name = JobName "sf-ci-functional-minimal"
                            , parent = Nothing
                            , nodeset = Nothing
                            , branches = []
                            , dependencies =
                                [ JobName "sf-rpm-build" ]
                            }
                        )
                    , PJJob
                        ( Job
                            { name = JobName "sf-ci-functional-allinone"
                            , parent = Nothing
                            , nodeset = Nothing
                            , branches = []
                            , dependencies =
                                [ JobName "sf-rpm-build" ]
                            }
                        )
                    , PJJob
                        ( Job
                            { name = JobName "wait-for-changes-ahead"
                            , parent = Nothing
                            , nodeset = Nothing
                            , branches = []
                            , dependencies =
                                [ JobName "sf-ci-functional-minimal"
                                , JobName "sf-ci-functional-allinone"
                                ]
                            }
                        )
                    ]
                }
            ]
        }
    )
]